test/e2e/resource_template_test.go

```diff
@@ -94,6 +94,52 @@ spec:
 		})
 }
 
+func (s *ResourceTemplateSuite) TestResourceTemplateWithArtifact() {
+	s.Given().
+		Workflow(`
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: k8s-resource-tmpl-with-artifact-
+spec:
+  serviceAccount: argo
+  entrypoint: main
+  templates:
+    - name: main
+      serviceAccountName: argo
+      inputs:
+        artifacts:
+        - name: kubectl
+          path: /tmp/manifest.yaml
+          raw:
+            data: |
+              apiVersion: v1
+              kind: Pod
+              metadata:
+                generateName: k8s-pod-resource-
+              spec:
+                serviceAccountName: argo
+                containers:
+                - name: argosay-container
+                  image: argoproj/argosay:v2
+                  command: ["/argosay"]
+                restartPolicy: Never
+      resource:
+        action: create
+        successCondition: status.phase == Succeeded
+        failureCondition: status.phase == Failed
+        manifestFrom:
+          path: /tmp/manifest.yaml
```

Comment:
I don't see `manifestFrom` being used anywhere in the implementation so I don't think path other than `/tmp/manifest.yaml` works.